create-serverless-service/test/test.js

```diff
@@ -70,11 +70,11 @@ describe('create-serverless-service', function() {
     //set up for default valid values to pass into handler()
     beforeEach(function(){
       event = {
-        "stage" : "test",
-        "headers" : {
-          "Authorization" : "fr1end$hip_1s_mAg1c"
-        },
+        "stage" : "test"
```

Comment:
There is a syntax error at line 73 for this object which is causing the the test suite to fail. Include a "," at the end of that line and that should do the trick; re-test the file for verification.

Reply:
Fixed the issue